cmd/handler.go

```diff
@@ -112,6 +112,13 @@ var handlerCmd = &cobra.Command{
 		} else {
 			ctx.Warn("AMQP is not enabled in your configuration")
 		}
+
+		if viper.GetString("handler.device-builtin") != "" {
```

Comment:
There is this option. Add permanent attributes in the binary can ensure some attribute keys will always be accepted.
Changing the config to `--extra-devices-attributes` or `--devices-extra-attributes` ?